AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/IncomingAsyncSubscriptionEventPublisher.swift

```diff
@@ -79,55 +82,58 @@ final class IncomingAsyncSubscriptionEventPublisher: Cancellable {
     func onCreateListenerHandler(event: Event) {
         log.verbose("onCreateListener: \(event)")
```

Comment:
It seems like we're repeating the guts of this method in `onCreateListenerHandler`, `onUpdateListenerHandler`, and `onDeleteListenerHandler`?

Reply:
Thanks for calling this out.  I struggled at first to figure out how to do this better -- ended up with an update.  Feel free to suggest something better.